financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,167 @@
+
+== Before You Begin
+
+* Create an Anypoint Platform account xref:rpa-home::automation-userrolesandpermissions-anypointrpapermissions.adoc[and assign it the `RPA Automations Manager` and `RPA Bots Manager` permissions].
+* Download the latest version of MuleSoft RPA Manager, RPA Builder, and RPA Bot.
+* Ensure you have access to your AMS and its database metadata.
+* Ensure you have access to the Salesforce Financial Services Cloud organization to which your AMS data will be loaded.
+* Connect the RPA Bot that is built for this solution to RPA Manager.
+* Download the “AMS_to_FSC.crpa” file to the same device on which RPA Builder is installed.
+
+== Workflow Overview
+
+. Create a project in RPA Manager.
+. Import and adapt the .crpa file in RPA Builder.
+. Publish or Deploy the process in RPA Manager.
+
+== Creating a Project in RPA Manager
+
+Detailed instructions for creating a new project in RPA Manager are available in the xref:rpa-manager::processautomation-manage.adoc#create-an-automation-project[Create an Automation Project] section of the Managing Automations page. If a new category is needed when creating a project, follow the instructions in the xref:rpa-manager::processautomation-prepare-project-category.adoc#create-a-category[Create a Category] section of the Manage Categories page.
+
+Once you create the project, xref:rpa-manager::processautomation-develop.adoc#build-a-process[move it directly to the Build phase] without making any changes.
+
+== Importing the CRPA File to RPA Builder
+
+. Log in to RPA Builder on the device on which the process runs.
+. On the xref:rpa-builder::ui-overview.adoc[Repository tab in the Explorer pane] in RPA Builder, click the name of the process you created in RPA Manager.
+. Click *File* on the xref:rpa-builder::ui-overview.adoc#menu-bar[Menu Bar], then click *Import Project*.
+. Select the .crpa file you downloaded in Exchange. The project opens in the model pane and includes three activities, which you must adapt to your AMS.
+
+== Adapting the CRPA File to Your AMS
+
+. Double-click each activity on the model pane to open the workflow.
+. Compare the shown process with your process and adapt the action steps accordingly. See How to adapt the CRPA file for an example.
+. Save and upload the project to RPA Manager.
+
+== Transferring Project to the Test Phase and Testing It
+
+Now that you have adapted the file to your AMS, xref:rpa-manager::processautomation-deploy.adoc#test-a-process-automation[follow these steps] to transfer the project to the Test phase and test it.
+
+== Transferring the Project to the Production Phase
+
+After successfully testing the project, xref:rpa-manager::processautomation-deploy.adoc[transfer the project to the Production phase]. Once transferred, do one of the following:
```

Comment:
Remove link, because the phase transition is part of each of the two options.
(We still need to clarify whether to explain both options or choose one. )